src/mouse_events.cpp

```diff
@@ -292,37 +293,49 @@ void mouse_handler::mouse_motion(int x, int y, const bool browse, bool update, m
 	if ( (!selected_hex_.valid()) && un && current_paths_.destinations.empty() &&
 			!gui().fogged(un->get_location()))
 	{
-		if (un->side() == side_num_) {
-			//unit is on our team, show path if the unit has one
-			const map_location go_to = un->get_goto();
-			if(board.map().on_board(go_to)) {
-				pathfind::marked_route route;
-				{ // start planned unit map scope
-					wb::future_map_if_active raii;
-					route = get_route(un.get(), go_to, current_team());
-				} // end planned unit map scope
-				gui().set_route(&route);
+		//   Only process unit if toggler not preventing unit
```

Comment:
Remove excessive indent in comment line.

Reply:
Will comply request.